stan/bernoulli/bernoulli.stan

```diff
@@ -9,7 +9,7 @@ transformed parameters {
   real logit_theta = logit(theta);
 }
 model {
-  theta ~ beta(1,1);
+  theta ~ beta(2, 2);
```

Comment:
I assume this is why the tests are failing? We test against the returned value of log_prob for this model

Reply:
That makes sense.  I can change that back.  I was trying to do minimal changes so that the flags all did something, but wound up just adding a regression example.

Reply:
Right now, I'm hung up on getting the paths right for the build-all.  I'm terrible at all this dev ops stuff.

Reply:
I think I fixed the python test paths in #22 if you merge in `main`